### ISSUE TITLE
a question about attack

### ISSUE THREAD

**Issue by liangbright** [qiliu08/frequency-refinement-defense#1]
central_class_to_background_loss = torch.sum(out_channel * opt_background_mask * pred_central_class_mask)
background_to_central_class_loss = torch.sum(out_channel * opt_central_class_mask * pred_background_mask)
loss = central_class_to_background_loss - background_to_central_class_loss

//loss.backward()

perturbed_im = image_to_optimize.data - (image_to_optimize.grad * pert_mul)

to reduce the segmentation accuracy, the overlap between central_class and background needs to be enlarged

(1) why is central_class_to_background_loss minimized?

(2) have you tried PGD-based adversarial attack ?
